ambari-server/src/main/java/org/apache/ambari/server/orm/dao/HostInfoSummaryDTO.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.ambari.server.orm.dao;
+
+public class HostInfoSummaryDTO {
```

Comment:
Can we just serialize this class back? Add some Jackson annotations to help with it?

Reply:
Here we want to get the data format as osType:osTypeCount, like:
centos6:10
debian5:2
This class will give us:
osType: centos
osTypeCount: 12
..
So I create another class HostInfoSummary to wrap it.